README.md

```diff
@@ -54,40 +54,46 @@ sudo mv kompose /usr/local/bin
 ## Bash completion
 Running this below command in order to benefit from bash completion
 
-```console
+```sh
 $ PROG=kompose source script/bash_autocomplete
 ```
 
 ## Building
 
 ### Building with `go`
 
-- You need `make`
-- You need `go` v1.6 or later.
-- If your working copy is not in your `GOPATH`, you need to set it accordingly.
+- `make`
```

Comment:
it looks like a requirements list, people would know what `make` is since they also see that `go` and `$GOPATH` is required.